dataaccess/tests/bundle-access.go

```diff
@@ -89,13 +89,6 @@ func (da DataAccessTester) testBundleCreateMissingRequired(t *testing.T) {
 	err = da.BundleCreate(da.ctx, bundle)
 	require.Error(t, err, errs.ErrFieldRequired)
 	bundle.GortBundleVersion = originalGortBundleVersion
-
-	// Description
```

Comment:
Unrelated: this is a test fix I made in response to removing the `description` requirement for bundles.